SKILL.tsv

```diff
@@ -5700,7 +5700,7 @@ SKILL_20151102_005699	$Afflicts stun at a 1.5% chance per attribute level for 3
 SKILL_20151102_005700	$Afflicts stun at a 2% chance per attribute level when performing a basic attack on an enemy with a [Blunt Instrument].
 SKILL_20151102_005701	$Enemies receive Holy damage equal to [Attribute Level] when using [Arcane Energy].
 SKILL_20151102_005702	$Healing Factor: Attack
-SKILL_20151102_005703	
+SKILL_20151102_005703	$Enemies traversing over [Healing Factor] tiles will incur continuous damage. Overall damage dealt scales with character's HP.
```

Comment:
No idea, I only know it's something put on the ground "Magic formation". Prolly circle actually since iirc they called magic circles in Mage Tower 'formations'